rental.rb

```diff
@@ -1,13 +1,17 @@
 class Rental
   attr_accessor :date, :book, :person
 
+  @assigned_ids = []
+
   def initialize(date, book, person)
     @date = date
     @book = book
     @person = person
-
-    # Update associations
     book.rentals << self
     person.rentals << self
```

Comment:
- [ ] Great effort working on your the `refactor your code` milestone, but currently i notice when i tried to create rentals there is an error that popped up which has hindered me from been able to create rentals. I would kindly advice you work on it and be sure to have a working project where user will be able to add rentals and also perform other activity.

Screenshot 👇 

![image](https://github.com/StevenWafeek/OOP_school_library/assets/78634841/f601f660-c6d8-417f-8b42-f4e457616cc9)